### ISSUE TITLE
yarn start > Failed to compile.

### ISSUE THREAD

**Issue by Bek81** [Shopify/slate#555]
Hi,

we created with "yarn" a new project and then replaced the basic theme with the "Modular" template (but we also tried with a basic theme). When we try to use the "yarn start" command we always get this issue:

Failed to compile.

undefined
../node_modules/@shopify/slate-tools/tools/webpack/static-files-glob.js
Module not found: Error: Can not resolve '__appstatic__' in '/ Applications / MAMP / htdocs / clemoonie / e-commerce / node_modules / @ shopify / slate-tools / tools / webpack'
  @ ../node_modules/@shopify/slate-tools/tools/webpack/static-files-glob.js 31: 0-38
  @ multi ../node_modules/@shopify/slate-tools/tools/webpack/hot-client.js ../node_modules/@shopify/slate-tools/tools/webpack/static-files-glob.js

The files are not compiled in dist and as a result are not even uploaded to the online theme of the store shopify.

Can you help us? please....

**Comment by ninjaaa** [comment]
Hi @Bek81 .  I had exactly the same issue earlier today, except 'appvendors' was the module that could not be found.  I was attempting to use the Shopify Skeleton theme.

If you look at the file that is kicking up the error - _../node_modules/@shopify/slate-tools/tools/webpack/static-files-glob.js_ - the first few lines say...

> // This is used as an entry point in webpack.base.conf.js. It dynamically
> // require all liquid and json files from the user's directory (except theme.liquid)
> // as well as everything inside the src/assets/vendors directory.

Looking further down the file at line 31, there is a required module that needs to be present.  Note the last line above _"as well as everything inside the src/assets/vendors directory."_

In my case, I noticed that there wasn't a "vendors" directory.  I resolved it was by adding the missing "vendors" directory into the following path... _my-shopify-theme/src/assets_.  Once the vendors directory was present I was able to successfully compile.  I would hazard a guess that if you added a directory called "static" into _your-shopify-theme/src/assets_ you'll be able to compile successfully.

Good luck.
R
